unity/unity-sources/Runtime/Mono/MonoFunctions.h

```diff
@@ -344,7 +344,7 @@ DO_API(void, mono_dl_fallback_unregister, (MonoDlFallbackHandler * handler))
 
 #endif
 
-typedef UNUSED_SYMBOL void(*vprintf_func)(const char* msg, va_list args);
+typedef UNUSED_SYMBOL int(*vprintf_func)(const char* msg, va_list args);
```

Comment:
Is this something we plan on keeping?  We probably shouldn't mismatch our function signatures - even if it shouldn't cause a problem.  We could write a small wrapper for `vprintf` so we have a default that matches our vprintf_func signature.

Reply:
I think that we should keep this, as it will be useful in the future for logging. @UnityAlex: what do you think about the wrapper suggestion?

Reply:
I was making the declaration match mono's side. AFAIK it's Unity's definition that needs to match this? The C++ vprintf returns an int as well. The x86 failure vs x64 building fine is interesting though.

Reply:
Oh, then maybe this was always wrong in the MonoFunctions.h here. I think this is fine then.